### PR TITLE
Remove resizable

### PR DESCRIPTION
**Pull request by cquiroz**
I'm removing the resizable area between the tree in the central area. It is not very usable and doesn't add much benefit while complicating the code

The PR ended up being fairly large as several refactoring and cleanups are included.
A couple of bugfixes are included too

### CODE REVIEW

**Comment by cquiroz** [comment]
I was expecting this PR to reduce the js size 😲

**Review by toddburnside — APPROVED** [review]
LGTM

**Comment by toddburnside** [comment]
> I was expecting this PR to reduce the js size 😲

That's strange. Is JS size just random?

File: common/src/main/scala/explore/syntax/ui/package.scala [hunk 1]
```diff
@@ -23,8 +23,9 @@ import scala.scalajs.js
 import scala.scalajs.js.UndefOr
 
 extension (self: Window)
+  // Keep this as def to take the value window.innerWidth at the current time
   def canFitTwoPanels: Boolean =
-    self.innerWidth <= Constants.TwoPanelCutoff
+    self.innerWidth > Constants.TwoPanelCutoff
```
[the remark this answers is not in the document]
Comment (cquiroz) [reply common/src/main/scala/explore/syntax/ui/package.scala:28]: Right, there was another similar function that was correct, this was used in one place and the other in others 🤷

**Pull request merged**